docs/pages/setup/admin/adding-nodes.mdx

```diff
@@ -50,7 +50,7 @@ it can also generate a new token for you). In the following example a new token
 is created with a TTL of 5 minutes:
 
 ```code
-$ tctl nodes add --ttl=5m --roles=node,proxy --token=secret-value
+$ tctl nodes add --ttl=5m --roles=node --token=secret-value
```

Comment:
While we're here, should we just remove `--token=secret-value`?

People are terrible at choosing good secrets. So it's both easier and more secure (IMO) to let Teleport choose it.

Reply:
Yeah, I can agree with that.